tools/etl/tg-jdbc-driver/tg-jdbc-examples/src/main/java/com/tigergraph/jdbc/examples/ConnectionPool.java

```diff
@@ -12,35 +12,45 @@
 import java.sql.Statement;
 
 /**
+ * Running on TigerGraph V3.4
```

Comment:
Please remove "Running on TigerGraph V3.4", it's supposed to be compatible with all versions.